source/intro.tex

```diff
@@ -788,10 +795,14 @@
 \begin{itemize}
 \item
 a preprocessing translation unit containing
-a \tcode{\#error} preprocessing directive\iref{cpp.error} or
+a \tcode{\#error} preprocessing directive\iref{cpp.error},  %JMB: this last comma was a mistake in P2900, it was not in green.
```

Comment:
In general, fixing commas and such can be done on-the-fly without much ado.

Please remove the "JMB" comment; we don't want that in the merged .tex.

Reply:
Yes, I just wanted to make sure additional eyes saw any deviations that occured relative to P2900's cwg-reviewed text.  (And also yes, we shouldn't merge this until all of the JMB comments have been looked at and removed).